### ISSUE TITLE
Media Alt Text should be required

### ISSUE THREAD

**Issue by tadjh** [dggpoliticalaction/org-repo#459]
Alt text is required by Next.js Image Component.

No alt text is bad for accessibility.

Apparently Alt Text helps with Picture SEO.

Our data model should enforce that.

Old media needs backfilled alt text in order for this not to break production.

The migration file should handle backfilling.

We should backfill with empty string so screen readers can skip reading out the filename due to missing alt text.
